sdk/python/kfp/components/_components.py

```diff
@@ -163,6 +165,16 @@ def _load_component_spec_from_component_text(text) -> ComponentSpec:
     component_dict = load_yaml(text)
     component_spec = ComponentSpec.from_dict(component_dict)
 
+    if isinstance(component_spec.implementation, ContainerImplementation) and (
+        component_spec.implementation.container.command is None):
+        warnings.warn(
+            'Container component must specify command to be compatible with '
+            'emissary executor, which will be the default executor for KFP.'
+            'https://github.com/kubeflow/pipelines/issues/5718',
```

Comment:
https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/ this is a better reference